### PR TITLE
ci: Run tests before release

### PR DESCRIPTION
**Pull request by flaviodsr**
Include test jobs, the same that are executed by the CI, as part of
the release workflow.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -7,8 +7,13 @@ on:
       - "v*"
 
 jobs:
+  test:
+    uses: flaviodsr/fuseml/.github/workflows/ci.yml@main
```
Comment (jsuchome) [inline .github/workflows/release.yml:11]: this should point to main repo

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -98,3 +104,7 @@ jobs:
       - name: Push image
         run: |
           make docker-push
+
+  fetch-installer:
+    uses: flaviodsr/fuseml/.github/workflows/get-fuseml-installer.yml@main
```
Comment (jsuchome) [inline .github/workflows/release.yml:109]: Same here

**Review by jsuchome — CHANGES_REQUESTED** [review]
see comments

File: .github/workflows/release.yml (hunk 1 above)
Comment (stefannica) [reply .github/workflows/release.yml]: fixed

File: .github/workflows/release.yml (hunk 2 above)
Comment (stefannica) [reply .github/workflows/release.yml]: fixed

**Review by stefannica — APPROVED** [review]

**Review by jsuchome — APPROVED** [review]

**Pull request merged**